cmd/generate/codeowners/codeowners.go

```diff
@@ -139,6 +143,7 @@ pizza generate codeowners . --config /path/to/.sauced.yaml
 
 	cmd.PersistentFlags().IntP("range", "r", 90, "The number of days to analyze commit history (default 90)")
 	cmd.PersistentFlags().Bool("owners-style-file", false, "Generate an agnostic OWNERS style file instead of CODEOWNERS.")
+	cmd.PersistentFlags().StringP("output-path", "o", "./", "Directory to create the output file.")
```

Comment:
That makes sense–

Made the following changes:

1) Default the `outputPath` argument to be the `path` argument (target directory). This will ensure that if the `outputFlag` is not supplied, the base location will be at the root of the repo path which is a mandatory argument

2) If the `outputFlag` is provided, it will generate the file in that location, distinct from the repo path that is provided.